README.md

```diff
@@ -1,113 +1,106 @@
+Once the webapp is launched, navigate to your browser settings and then to site settings and allow for insecure content.
+Since the stream and chat from the AWS servers is sent over http without encryption, this is needed for the stream requests and chat requests to the AWS server to work.
+Alternatively, you can click on the lock icon on the address bar and navigate to the site settings there.
 
-## Learn More
+### Enabling Third-Party Cookies
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Since Google Sign-In requires third-party cookies in order to function correctly, ensure that your browser is not blocking third-party cookies. To do this in Chrome, go to Settings > Privacy and Security > Cookies and Site Data > Block third-party cookies. This option should be disabled. Having it enabled may result in the Google Sign-In button not rendering at all.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## Deployment Instructions
 
-### Code Splitting
+### Cloning Github Repositories
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+#### Cloning Twijj
 
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
+`git clone https://github.com/josh-velasquez/Twijj` <br>
+Navigate to the client folder<br>
+`npm install`<br>
+To run the application<br>
+`npm start`<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+#### Cloning Twijj Chat Server
 
-### `npm run build` fails to minify
+`git clone https://github.com/josh-velasquez/TwijjChatServer`<br>
+Navigate to folder<br>
+`npm install`<br>
+To run the application<br>
+`npm start`<br>
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+#### Cloning Twijj Stream Server
 
-## Building the project
+`git clone https://github.com/josh-velasquez/TwijjStreamServer`<br>
+Navigate to rtmpserver folder<br>
+`npm install`<br>
+To run the application<br>
+`npm start`<br>
 
-- Run npm install on all three folders
-  - .gitignore excludes the node_modules
+<br>
+<br>
 
-# Setting up OBS
+# Live Deployment
 
-- Create a scene
-- Select the video input
-- Go to settings and navigate to Stream
-  - Select Custom for Service
-  - Enter rtmp://localhost/live for Server
-  - Enter the ID of the stream (located at the client browser url)
+## Deploying To Build To Firebase
```

Comment:
Should this be "Deploying a Build to Firebase"?

Reply:
Updated :D